src/db/migrations/20250109140803_add_churn_prevention_email_sent.js

```diff
@@ -0,0 +1,25 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+export function up (knex) {
+  return knex.schema.table("subscribers", table => {
+    table.boolean("churn_prevention_email_sent").defaultTo(false);
```

Comment:
Should it maybe be a date instead? Since users may potentially churn again next year? Or are we assuming we'll have a better solution by then? :sweat_smile:

Reply:
Hmm interesting.. If there's another strategy, I don't think it's going to be sending the same email. For consistency sake, I think it's probably best to use boolean but that's an interesting point.

Reply:
Actually, we do have precedent for using ts, so I'll change it to using ts and check for null instead of `false`